5e-SRD-Monsters.json

```diff
@@ -7432,8 +7645,15 @@
         ]
       },
       {
-        "name": "Relentless (Recharges after a Short or Long Rest)",
-        "desc": "If the boar takes 7 damage or less that would reduce it to 0 hit points, it is reduced to 1 hit point instead."
+        "name": "Relentless",
+        "desc": "If the boar takes 7 damage or less that would reduce it to 0 hit points, it is reduced to 1 hit point instead.",
+        "usage": {
```

Comment:
For the two types of recharge, would it make more sense to structure it like so?
```json
"recharge": {
  "type": "dice",
  "dice": "1d6",
  "values: [5,6]
}
```

```json
"recharge": {
  "type": "rest",
  "rest_types": [
    "short",
    "long
  ]
}
```

Reply:
Yes, or I can shortcut it like this:

    "dice": "1d6",
    "min_value": 4

What do you prefer?

Reply:
Fair enough!